README.md

```diff
@@ -1,5 +1,50 @@
 # Project outline
 
+## FastSpeech2 setup
+
+1. Install Anaconda (<https://www.anaconda.com/products/distribution#Downloads>)
+2. Create a virtual environment and then activate it. Name (`-n`) can be whatever.
+
+   ```bash
+   conda create -n fastspeech python=3.7.4
+   conda activate fastspeech
+   ```
+
+3. Pull the latest remote and update submodule
+   
+   ```bash
+   git pull origin
+   git submodule update --init
+   ```
+
+4. Enter the FastSpeech2 folder and install required packages
+   
+   ```bash
+   cd NEU-tts-2022
+   pip install -r requirements.txt
+   ```
+
+   Install PyTorch
+   
+   ```bash
+   # Intel MacBook
```

Comment:
Can these be applied to other Intel devices too?

Reply:
Yeah I'm using a Windows notebook rn. Since you guys are using MacBook I only put the MacOS commands here